diff -ur woocommerce-10.0.2/includes/admin/helper/class-wc-helper.php woocommerce-10.0.2-patched/includes/admin/helper/class-wc-helper.php
--- woocommerce-10.0.2/includes/admin/helper/class-wc-helper.php	2025-06-23 15:46:28.000000000 +0100
+++ woocommerce-10.0.2-patched/includes/admin/helper/class-wc-helper.php	2025-07-14 18:20:42.905271080 +0100
@@ -1686,6 +1686,8 @@
 	 * phpcs:ignore Squiz.Commenting.FunctionCommentThrowTag.Missing -- As we wrap the throw in a try/catch.
 	 */
 	public static function get_product_usage_notice_rules() {
+        return []; // Stop calling home when we don't need to.
+        
 		$cache_key = '_woocommerce_helper_product_usage_notice_rules';
 		$data      = get_transient( $cache_key );
 		if ( false !== $data ) {
Only in woocommerce-10.0.2-patched/includes/admin/helper: class-wc-helper.php.orig
diff -ur woocommerce-10.0.2/includes/class-woocommerce.php woocommerce-10.0.2-patched/includes/class-woocommerce.php
--- woocommerce-10.0.2/includes/class-woocommerce.php	2025-07-14 09:28:08.000000000 +0100
+++ woocommerce-10.0.2-patched/includes/class-woocommerce.php	2025-07-14 18:20:42.905271080 +0100
@@ -297,7 +297,18 @@
 		add_action( 'init', array( $this, 'add_image_sizes' ) );
 		add_action( 'init', array( $this, 'load_rest_api' ) );
 		if ( $this->is_request( 'admin' ) || ( $this->is_rest_api_request() && ! $this->is_store_api_request() ) || ( defined( 'WP_CLI' ) && WP_CLI ) ) {
-			add_action( 'init', array( 'WC_Site_Tracking', 'init' ) );
+            // Even with site usage tracking disabled, Woo still puts loads of tracking stuff into the browser.
+            // Let's get rid of that here.
+			// add_action( 'init', array( 'WC_Site_Tracking', 'init' ) );
+            // We need a stub so that some secondary scripts don't generate errors.
+            add_action( 'admin_footer', function(){
+                echo '<script type="text/javascript">';
+                echo 'window.wcTracks = window.wcTracks || {};';
+                echo 'window.wcTracks.isEnabled = false;';
+                echo 'window.wcTracks.validateEvent = () => {};';
+                echo 'window.wcTracks.recordEvent = () => {};';
+                echo '</script>';
+            });
 		}
 		add_action( 'switch_blog', array( $this, 'wpdb_table_fix' ), 0 );
 		add_action( 'activated_plugin', array( $this, 'activated_plugin' ) );
@@ -671,8 +682,8 @@
 		include_once WC_ABSPATH . 'includes/class-wc-logger.php';
 		include_once WC_ABSPATH . 'includes/queue/class-wc-action-queue.php';
 		include_once WC_ABSPATH . 'includes/queue/class-wc-queue.php';
-		include_once WC_ABSPATH . 'includes/admin/marketplace-suggestions/class-wc-marketplace-updater.php';
-		include_once WC_ABSPATH . 'includes/admin/class-wc-admin-marketplace-promotions.php';
+		// include_once WC_ABSPATH . 'includes/admin/marketplace-suggestions/class-wc-marketplace-updater.php';
+		// include_once WC_ABSPATH . 'includes/admin/class-wc-admin-marketplace-promotions.php';
 		include_once WC_ABSPATH . 'includes/blocks/class-wc-blocks-utils.php';
 
 		/**
@@ -850,6 +861,22 @@
 		 */
 		do_action( 'before_woocommerce_init' ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingSinceComment
 
+        // Force some options to no and autoload them.
+        // 'woocommerce_feature_order_attribution_enabled',
+        $options_to_no = [
+            'woocommerce_allow_tracking',
+            'marketplace_suggestions',
+            'woocommerce_show_marketplace_suggestions',
+            'woocommerce_feature_remote_logging_enabled',
+            'woocommerce_feature_reactify-classic-payments-settings_enabled',
+            'woocommerce_feature_blueprint_enabled'
+        ];
+        foreach( $options_to_no as $option_name ) {
+            if( filter_var( get_option( $option_name, false ),  FILTER_VALIDATE_BOOLEAN ) !== false ) {
+                update_option( $option_name, 'no', true );
+            }
+        }
+
 		// Set up localisation.
 		$this->load_plugin_textdomain();
 
Only in woocommerce-10.0.2-patched/includes: class-woocommerce.php.orig
diff -ur woocommerce-10.0.2/includes/tracks/class-wc-site-tracking.php woocommerce-10.0.2-patched/includes/tracks/class-wc-site-tracking.php
--- woocommerce-10.0.2/includes/tracks/class-wc-site-tracking.php	2024-11-13 20:17:00.000000000 +0000
+++ woocommerce-10.0.2-patched/includes/tracks/class-wc-site-tracking.php	2025-07-14 18:20:42.913271223 +0100
@@ -17,6 +17,9 @@
 	 * @return bool
 	 */
 	public static function is_tracking_enabled() {
+        // Never use remote logging
+        return false;
+        
 		/**
 		 * Don't track users if a filter has been applied to turn it off.
 		 * `woocommerce_apply_tracking` will be deprecated. Please use
diff -ur woocommerce-10.0.2/woocommerce.php woocommerce-10.0.2-patched/woocommerce.php
--- woocommerce-10.0.2/woocommerce.php	2025-07-14 09:28:08.000000000 +0100
+++ woocommerce-10.0.2-patched/woocommerce.php	2025-07-14 18:20:42.841269938 +0100
@@ -65,3 +65,18 @@
 if ( class_exists( \Automattic\Jetpack\Connection\Rest_Authentication::class ) ) {
 	\Automattic\Jetpack\Connection\Rest_Authentication::init();
 }
+
+// START : wpatcher
+add_filter(
+	'plugin_row_meta',
+	function ( $links, $file ) {
+		if ( $file == 'woocommerce/woocommerce.php' ) {
+			$links['wpatcher'] = '<span style="background-color: purple; color: white; padding:0.2em 0.5em; font-weight: 700; border-radius: 0.3em">PATCHED : 2025-07-14</span>';
+		}
+
+		return $links;
+	},
+	10,
+	2
+);
+// END : wpatcher
