diff -ur woocommerce-10.1.0/includes/admin/helper/class-wc-helper.php woocommerce-10.1.0-patched/includes/admin/helper/class-wc-helper.php
--- woocommerce-10.1.0/includes/admin/helper/class-wc-helper.php	2025-06-23 15:46:28.000000000 +0100
+++ woocommerce-10.1.0-patched/includes/admin/helper/class-wc-helper.php	2025-08-13 19:44:38.886825566 +0100
@@ -1686,6 +1686,8 @@
 	 * phpcs:ignore Squiz.Commenting.FunctionCommentThrowTag.Missing -- As we wrap the throw in a try/catch.
 	 */
 	public static function get_product_usage_notice_rules() {
+        return []; // Stop calling home when we don't need to.
+        
 		$cache_key = '_woocommerce_helper_product_usage_notice_rules';
 		$data      = get_transient( $cache_key );
 		if ( false !== $data ) {
diff -ur woocommerce-10.1.0/includes/class-woocommerce.php woocommerce-10.1.0-patched/includes/class-woocommerce.php
--- woocommerce-10.1.0/includes/class-woocommerce.php	2025-08-12 09:58:12.000000000 +0100
+++ woocommerce-10.1.0-patched/includes/class-woocommerce.php	2025-08-13 19:44:38.894825690 +0100
@@ -286,7 +286,7 @@
 
 		add_action( 'plugins_loaded', array( $this, 'on_plugins_loaded' ), -1 );
 		add_action( 'plugins_loaded', array( $this, 'init_customizer' ) );
-		add_action( 'plugins_loaded', array( $this, 'init_jetpack_connection_config' ), 1 );
+		// add_action( 'plugins_loaded', array( $this, 'init_jetpack_connection_config' ), 1 );
 		add_action( 'admin_notices', array( $this, 'build_dependencies_notice' ) );
 		add_action( 'after_setup_theme', array( $this, 'setup_environment' ) );
 		add_action( 'after_setup_theme', array( $this, 'include_template_functions' ), 11 );
@@ -297,7 +297,18 @@
 		add_action( 'init', array( $this, 'add_image_sizes' ) );
 		add_action( 'init', array( $this, 'load_rest_api' ) );
 		if ( $this->is_request( 'admin' ) || ( $this->is_rest_api_request() && ! $this->is_store_api_request() ) || ( defined( 'WP_CLI' ) && WP_CLI ) ) {
-			add_action( 'init', array( 'WC_Site_Tracking', 'init' ) );
+            // Even with site usage tracking disabled, Woo still puts loads of tracking stuff into the browser.
+            // Let's get rid of that here.
+			// add_action( 'init', array( 'WC_Site_Tracking', 'init' ) );
+            // We need a stub so that some secondary scripts don't generate errors.
+            add_action( 'admin_footer', function(){
+                echo '<script type="text/javascript">';
+                echo 'window.wcTracks = window.wcTracks || {};';
+                echo 'window.wcTracks.isEnabled = false;';
+                echo 'window.wcTracks.validateEvent = () => {};';
+                echo 'window.wcTracks.recordEvent = () => {};';
+                echo '</script>';
+            });
 		}
 		add_action( 'switch_blog', array( $this, 'wpdb_table_fix' ), 0 );
 		add_action( 'activated_plugin', array( $this, 'activated_plugin' ) );
@@ -674,8 +685,8 @@
 		include_once WC_ABSPATH . 'includes/class-wc-logger.php';
 		include_once WC_ABSPATH . 'includes/queue/class-wc-action-queue.php';
 		include_once WC_ABSPATH . 'includes/queue/class-wc-queue.php';
-		include_once WC_ABSPATH . 'includes/admin/marketplace-suggestions/class-wc-marketplace-updater.php';
-		include_once WC_ABSPATH . 'includes/admin/class-wc-admin-marketplace-promotions.php';
+		// include_once WC_ABSPATH . 'includes/admin/marketplace-suggestions/class-wc-marketplace-updater.php';
+		// include_once WC_ABSPATH . 'includes/admin/class-wc-admin-marketplace-promotions.php';
 		include_once WC_ABSPATH . 'includes/blocks/class-wc-blocks-utils.php';
 
 		/**
@@ -853,6 +864,23 @@
 		 */
 		do_action( 'before_woocommerce_init' ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingSinceComment
 
+        // Force some options to no and autoload them.
+        // 'woocommerce_feature_order_attribution_enabled',
+        $options_to_no = [
+            'woocommerce_allow_tracking',
+            'marketplace_suggestions',
+            'woocommerce_show_marketplace_suggestions',
+            'woocommerce_feature_remote_logging_enabled',
+            'woocommerce_feature_reactify-classic-payments-settings_enabled',
+            'woocommerce_feature_blueprint_enabled',
+            'woocommerce_feature_point_of_sale_enabled'
+        ];
+        foreach( $options_to_no as $option_name ) {
+            if( filter_var( get_option( $option_name, false ),  FILTER_VALIDATE_BOOLEAN ) !== false ) {
+                update_option( $option_name, 'no', true );
+            }
+        }
+
 		// Set up localisation.
 		$this->load_plugin_textdomain();
 
Only in woocommerce-10.1.0-patched/includes: class-woocommerce.php.orig
diff -ur woocommerce-10.1.0/includes/tracks/class-wc-site-tracking.php woocommerce-10.1.0-patched/includes/tracks/class-wc-site-tracking.php
--- woocommerce-10.1.0/includes/tracks/class-wc-site-tracking.php	2024-11-13 20:17:00.000000000 +0000
+++ woocommerce-10.1.0-patched/includes/tracks/class-wc-site-tracking.php	2025-08-13 19:44:38.894825690 +0100
@@ -17,6 +17,9 @@
 	 * @return bool
 	 */
 	public static function is_tracking_enabled() {
+        // Never use remote logging
+        return false;
+        
 		/**
 		 * Don't track users if a filter has been applied to turn it off.
 		 * `woocommerce_apply_tracking` will be deprecated. Please use
diff -ur woocommerce-10.1.0/src/Internal/Admin/Settings/Payments.php woocommerce-10.1.0-patched/src/Internal/Admin/Settings/Payments.php
--- woocommerce-10.1.0/src/Internal/Admin/Settings/Payments.php	2025-07-29 08:34:58.000000000 +0100
+++ woocommerce-10.1.0-patched/src/Internal/Admin/Settings/Payments.php	2025-08-13 19:44:38.914825997 +0100
@@ -80,9 +80,11 @@
 		$payment_providers = array();
 
 		// Only include suggestions if the requesting user can install plugins.
-		if ( current_user_can( 'install_plugins' ) ) {
-			$suggestions = $this->providers->get_extension_suggestions( $location, self::SUGGESTIONS_CONTEXT );
-		}
+        // 2025-08-10 PF. Stop making outbound connections to Automattic
+		// if ( current_user_can( 'install_plugins' ) ) {
+        // $suggestions = $this->providers->get_extension_suggestions( $location, self::SUGGESTIONS_CONTEXT );
+		// }
+        
 		// If we have preferred suggestions, add them to the providers list.
 		if ( ! empty( $suggestions['preferred'] ) ) {
 			// Sort them by priority, ASC.
Only in woocommerce-10.1.0-patched/src/Internal/Admin/Settings: Payments.php.orig
diff -ur woocommerce-10.1.0/vendor/automattic/jetpack-connection/src/class-plugin.php woocommerce-10.1.0-patched/vendor/automattic/jetpack-connection/src/class-plugin.php
--- woocommerce-10.1.0/vendor/automattic/jetpack-connection/src/class-plugin.php	2025-06-23 15:46:28.000000000 +0100
+++ woocommerce-10.1.0-patched/vendor/automattic/jetpack-connection/src/class-plugin.php	2025-08-13 19:44:38.922826120 +0100
@@ -47,7 +47,7 @@
 		$this->slug = $slug;
 
 		// Initialize Users_Connection_Admin
-		$this->users_connection_admin = new Users_Connection_Admin();
+		// $this->users_connection_admin = new Users_Connection_Admin();
 	}
 
 	/**
diff -ur woocommerce-10.1.0/woocommerce.php woocommerce-10.1.0-patched/woocommerce.php
--- woocommerce-10.1.0/woocommerce.php	2025-08-12 09:58:12.000000000 +0100
+++ woocommerce-10.1.0-patched/woocommerce.php	2025-08-13 19:44:38.898825750 +0100
@@ -65,3 +65,18 @@
 if ( class_exists( \Automattic\Jetpack\Connection\Rest_Authentication::class ) ) {
 	\Automattic\Jetpack\Connection\Rest_Authentication::init();
 }
+
+// START : wpatcher
+add_filter(
+	'plugin_row_meta',
+	function ( $links, $file ) {
+		if ( $file == 'woocommerce/woocommerce.php' ) {
+			$links['wpatcher'] = '<span style="background-color: purple; color: white; padding:0.2em 0.5em; font-weight: 700; border-radius: 0.3em">PATCHED : 2025-08-13</span>';
+		}
+
+		return $links;
+	},
+	10,
+	2
+);
+// END : wpatcher
